propertyestimator/properties/enthalpy.py

```diff
@@ -605,7 +605,7 @@ def get_default_simulation_workflow_schema(options=None):
         gas_protocols.equilibration_simulation.enable_pbc = False
 
         gas_protocols.production_simulation.ensemble = Ensemble.NVT
-        gas_protocols.production_simulation.steps = 1000000
+        gas_protocols.production_simulation.steps = 10000000
```

Comment:
Are we able to automatically extend these simulations until convergence is achieved? Does `.steps` just control the size of a simulation block in between convergence checks?

Reply:
> Does `.steps` just control the size of a simulation block in between convergence checks

Exactly this!